demo/wdio-cucumber.conf.js

```diff
@@ -14,7 +18,7 @@ export const config = {
   // Custom settings
   // ===============
   logLevel: 'info', // trace | debug | info | warn | error | silent
-  outputDir: './_results_',
+  outputDir: `${__dirname}/_results_`,
```

Comment:
Make sure this also works in windows
```suggestion
  outputDir: path.join(__dirname, '_results_'),
```